.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 4
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
Nice!

Reply:
Yea I'd never used the EditorConfig extension with VS Code (no need), but it works a treat. Just need to add the config, close and re-open VS Code and it even respects nesting of configs with different settings 👍